ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -165,7 +165,19 @@ if [[ "$PRE_PXE_LOADER" == "true" ]]; then
   echo "Modifying PXE loaders ..."
   for i in $HOSTS; do
     echo "Modifying PXE loader of server $i ..."
-    hammer -c /root/.hammer/cli.modules.d/foreman_$LAB.yml --verify-ssl false -u $LAB_CLOUD -p $PWD host update --name $i --operatingsystem "$FOREMAN_OS" --pxe-loader "PXELinux BIOS" --build 1
+    podman run \
+      -v /root/.hammer/cli.modules.d:/opt/hammer \
```

Comment:
I dont like to depend on files that we assume to be in the bastion host, couldnt we generate dynamically this files on `/tmp/foreman_$LAB.yml` when running the test instead?